src/bolos/bagl.c

```diff
@@ -92,7 +101,7 @@ unsigned long sys_bagl_hal_draw_bitmap_within_rect(
   size_t i, len, size;
   uint8_t buf[300 - 4]; /* size limit in io_seph_send */
   uint8_t header[4];
-
+  uint32_t character = get_character_from_bitmap(bitmap);
```

Comment:
could be skipped on NANOS, if this syscall is really called on this device?

Reply:
character is supposed to be always present in data, even for unsupported devices/api_levels.
This function will return 0, which is the expected behaviour in that case.